composer.json

```diff
@@ -51,23 +52,20 @@
             "client/dist"
         ],
         "project-files-installed": [
+            "behat.yml",
             "phpcs.xml",
-            "phpunit.xml"
+            "phpunit.xml",
+            "phpunit.xml.dist"
         ]
     },
     "minimum-stability": "dev",
     "prefer-stable": true,
-    "repositories": [
```

Comment:
Removing dependency due to refactor, this module was heavily dependent on `SwiftMailer` which has been dropped in SS5

Reply:
Bro.. fml 🙃 I think the craziest part was the number of trial and errors that took place to arrive at this approach - I switched approaches 3-4 times because of limitations 😬